docs/xls-73d/AMMClawback.md

```diff
@@ -0,0 +1,68 @@
+# AMMClawback
+
+<!--> Add link to github source after feature is merged into devnet. <-->
+[[Source]](https://github.com/XRPLF/rippled/pull/5142 "Source")
+
+{% partial file="/snippets/_ammclawback-disclaimer.md" /%}
+
+Claw back tokens issued by your account that have been deposited into an AMM.
```

Comment:
maybe give more detail:

Claw back tokens from specified holder that have been deposited into an AMM pool by the holder. If the issuer issues both assets in the AMM pool and flag tfClawTwoAssets is set, then the other asset will also be clawed back proportionally based on the current AMM pool proportion. Otherwise, the other asset will return to the holder proportionally.